manifest.json

```diff
@@ -3,18 +3,18 @@
     "id": "pterodactyl",
     "packaging_format": 1,
     "description": {
-        "en": "Pterodactyl package for YunoHost.",
-        "fr": "Pterodactyl pour YunoHost."
+        "en": "Pterodactyl is a gameserver management panel (Minecraft, ARK, Rust,...)",
+        "fr": "Pterodactyl est un panel de gestion de serveurs de jeux (Minecraft, ARK, Rust,...)"
     },
     "version": "0.7.11",
     "url": "https://pterodactyl.io/",
     "license": "free",
     "maintainer": {
-        "name": "liberodark",
-        "email": "liberodark@gmail.com"
+        "name": "Pterodactyl Team",
+        "email": "support@pterodactyl.io"
```

Comment:
Not sure about the mail address to use, is it correct ?

Reply:
No it's the YunoHost Packager name and email to provide :)
So your name if you want to be the packager of this app

Reply:
But as you corrected it in #27, I will close that PR :)